### ISSUE TITLE
10$: Warn when comma operator used with literals

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#724]
The following code should report a warning:
```js
let a = [];
a.include(('one', 'two'));
```

**Comment by strager** [comment]
- [ ] Distinguish comma expression from other expressions
- [ ] Report a diagnostic if a comma expression contains only literal expressions
